src/cljs/nr/gameboard/actions.cljs

```diff
@@ -108,18 +108,21 @@
       (build-report-url error)
       "');\">Report on GitHub</button></div>")))
 
+(defn ack-toast-impl ([id] (send-command "toast" {:id id})))
+(def ack-toast (memoize ack-toast-impl))
```

Comment:
This might grow big. Should we try to limit it?

Reply:
I spent a bit of time looking into trying to set up a fifo cache memoization strategy with clojure.core.memoize only to realize it's not cljs 👎🏻 - I actually think this memoization maybe isn't necessary/helpful - or if it would help it means the problem is different than what would be solved by giving a message with IDs (e.g. it'd mean unique toasts are being generated each message rather than the same message getting stuck).